src/server/cmd/pachctl/cmd/cmd.go

```diff
@@ -388,108 +388,6 @@ Currently "pachctl garbage-collect" can only be started when there are no active
 		}),
 	}
 
-	var from, to, namespace string
```

Comment:
I doubt you meant to remove all this migration stuff. I think you just need a merge

Reply:
It's actually an old implementation of migration from 1.4 to 1.5. Not really useful to anyone anymore and likely to confuse people.